catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/source/impl/SourceDescriptorImpl.java

```diff
@@ -95,7 +95,7 @@ public void setAvailable(boolean isAvailable) {
 
     @Override
     public Date getLastAvailabilityDate() {
-        return lastAvailableDate;
+        return (Date)lastAvailableDate.clone();
```

Comment:
Instead of `clone`, using the millis as you did elsewhere would be more efficient.